gen3.theanvil.io/manifest.json

```diff
@@ -7,32 +7,32 @@
     "autodeploy": "yes"
   },
   "versions": {
-    "arborist": "quay.io/cdis/arborist:deploy_anvil-staging",
+    "arborist": "quay.io/cdis/arborist:2020.02",
     "arranger-adminapi": "quay.io/cdis/arranger-server:master",
     "arranger-dashboard": "quay.io/cdis/arranger-dashboard:master",
     "aws-es-proxy": "abutaha/aws-es-proxy:0.8",
-    "fence": "quay.io/cdis/fence:4.2.1",
+    "fence": "quay.io/cdis/fence:2020.02",
     "fluentd": "fluent/fluentd-kubernetes-daemonset:v1.2-debian-cloudwatch",
-    "guppy": "quay.io/cdis/guppy:0.3.0",
-    "hatchery": "quay.io/cdis/hatchery:0.1.0",
-    "indexd": "quay.io/cdis/indexd:2.0.0",
-    "manifestservice": "quay.io/cdis/manifestservice:master",
-    "peregrine": "quay.io/cdis/peregrine:master",
-    "pidgin": "quay.io/cdis/pidgin:1.1.0",
-    "portal": "quay.io/cdis/data-portal:2.17.0",
+    "guppy": "quay.io/cdis/guppy:2020.02",
```

Comment:
Please verify the `etlMapping.yaml` and `portal/gitops.json` match what's working in staging - even though `guppy` is in the manifest - it's not actually running in production for some reason